lib/node_modules/@stdlib/regexp/to-json/docs/types/index.d.ts

```diff
@@ -0,0 +1,37 @@
+/*
+* @license Apache-2.0
+*
+* Copyright (c) 2022 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+// TypeScript Version: 2.0
+
+/**
+* Returns a JSON representation of a regexp object.
+*
+* @param regexp - regular expression to serialize
+* @returns JSON representation
+*
+* @example
+* var regex = new reRegExp('ab+c');
+* var json = toJSON( regex );
+* // returns {...}
+*/
+declare function toJSON( regex: RegExp ): any; // tslint-disable-line max-line-length
+
+
+// EXPORTS //
+
+export = toJSON;
```

Comment:
Do you have EditorConfig setup on your IDE? If so, adding a trailing newline to files should be automatic. If not, need to manually add.